docs/sphinx/source/drop.rst

```diff
@@ -0,0 +1,268 @@
+.. _drop:
+
+DROP - Data Reduction OPerator
+===============================================
+
+DROP (Data Reduction OPerator) is a comprehensive tool within the AQUA framework designed 
+to extract, process, and organize data from any dataset, with particular focus on climate 
+data reduction and archival.
```

Comment:
"with particular focus on climate data reduction and archival."not fully sure I understand, DROP works only for climate data.